bedrock/firefox/views.py

```diff
@@ -722,6 +725,30 @@ def get(self, *args, **kwargs):
 
         return super(NewView, self).get(*args, **kwargs)
 
+    def render_to_response(self, context, **response_kwargs):
+        # set experimental percentages per locale with this config
+        # e.g. EXP_CONFIG_FX_NEW=de:20,en-US:10,fr:25
+        # this would send 20% of de, 10% of en-US, and 25% of fr requests to the experiment page
+        # all other locales would be unaffected
+        redirect_percents = config('EXP_CONFIG_FX_NEW', default='', parser=DictOf(int))
+        skip_exp = 'automation' in self.request.GET
+        # only engage the experiment infra if some experiments are set
+        if redirect_percents and not skip_exp:
+            locale = l10n_utils.get_locale(self.request)
+            percent = redirect_percents.get(locale, 0)
+            if percent:
+                percent = percent / 100
+                print(percent)
+                if random() <= percent:
+                    response = HttpResponseRedirect(reverse('exp.firefox.new'))
```

Comment:
We probably need to make sure that URL params are preserved in the redirect here (sorry, this didn't cross my mind in our discussion yesterday). When I test locally params don't currently seem to be preserved.

Reply:
Good catch! I'll update that.

Reply:
Updated and added a test for params.